test/enterprise.test.ts

```diff
@@ -0,0 +1,146 @@
+import { Probot, ProbotOctokit } from "probot";
+import {
+  checkApproved,
+  setSinglePreviousReview,
+  verifyReviewDismissed,
+} from "./utils/review";
+import {
+  checkCrashStatus,
+  checkStartedStatus,
+  checkSuccessStatus,
+} from "./utils/status";
+import { prOpenedPayload, prReopenedPayload } from "./fixtures/payloads/basic";
+import { setConfigNotFound, setConfigToBasic } from "./utils/config";
+import { StatusCodes } from "http-status-codes";
+import { TEST_TIMEOUT } from "./utils/jest";
+import approvemanApp from "../src";
+import fs from "fs";
+import { getAppActorName } from "../src/utils/config";
+import { getGitHubAPIEndpoint } from "./utils/endpoint";
+import nock from "nock";
+import path from "path";
+
+/* eslint-disable */
+jest.setTimeout(TEST_TIMEOUT);
+/* eslint-enable */
+
+describe("Approveman tests", () => {
+  let probot: Probot;
+  let mockCert: string;
+  const defaultEnv = process.env;
+
+  beforeAll((done: jest.DoneCallback) => {
+    const mockCertLocation = path.join(__dirname, "fixtures/mock-cert.pem");
+    /* eslint-disable security/detect-non-literal-fs-filename */
```

Comment:
Codacy found an issue: ['jest' is not defined.](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6168221)